### PR TITLE
Add ShinkarenkoMaxim/simple-quiz and template ShinkarenkoMaxim/webapp-vanilla

### PR DESCRIPTION
**Pull request by ShinkarenkoMaxim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +42,4 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [PavelPolyakov/grammy-with-tests](https://github.com/PavelPolyakov/grammy-with-tests) - An example bot which is covered with tests.
 - [waptik/grammy-on-deta-template](https://github.com/waptik/grammy-on-deta-template) - A template to start a grammY bot and deploy it on [Deta](https://www.deta.sh). [ᴜsᴇ](https://t.me/GrammyOnDetaBot)
 - [askador/grammY-bot-template](https://github.com/askador/grammY-bot-template) - An example of a grammY bot file structure.
+- [ShinkarenkoMaxim/webapp-vanilla](https://github.com/ShinkarenkoMaxim/webapp-vanilla) - Telegram WebApp starter bundle with vanilla setup.
```
Comment (roj1512) [inline README.md:45]:
```suggestion
- [ShinkarenkoMaxim/webapp-vanilla](https://github.com/ShinkarenkoMaxim/webapp-vanilla) - Telegram Web App starter bundle with vanilla setup.
```

**Comment by KnorpelSenf** [comment]
@ShinkarenkoMaxim please also resolve the merge conflicts

File: README.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [KnorpelSenf/link-preview-bot](https://github.com/KnorpelSenf/link-preview-bot) - Generate link previews for all links in messages. [ᴜsᴇ](https://t.me/linkpreviewbot)
 - [dcdunkan/syntax-highlighter-bot](https://github.com/dcdunkan/syntax-highlighter-bot) - Syntax highlighting for code blocks in messages. [ᴜsᴇ](https://t.me/syntaxybot)
 - [bot-base/scan-tool-bot](https://github.com/bot-base/scan-tool-bot) - Scan QR codes with your camera. [ᴜsᴇ](https://t.me/ScanToolBot)
+- [ShinkarenkoMaxim/simple-quiz](https://github.com/ShinkarenkoMaxim/simple-quiz) - Simple quiz bot with Web App. [ᴜsᴇ](https://t.me/yaqb_bot)
```
Comment (roj1512) [inline README.md:33]:
```suggestion
- [ShinkarenkoMaxim/simple-quiz](https://github.com/ShinkarenkoMaxim/simple-quiz) - A simple quiz bot as Web App. [ᴜsᴇ](https://t.me/yaqb_bot)
```

File: README.md [hunk 3]
```diff
@@ -42,3 +43,4 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [PavelPolyakov/grammy-with-tests](https://github.com/PavelPolyakov/grammy-with-tests) - An example bot which is covered with tests.
 - [waptik/grammy-on-deta-template](https://github.com/waptik/grammy-on-deta-template) - A template to start a grammY bot and deploy it on [Deta](https://www.deta.sh). [ᴜsᴇ](https://t.me/GrammyOnDetaBot)
 - [askador/grammY-bot-template](https://github.com/askador/grammY-bot-template) - An example of a grammY bot file structure.
+- [ShinkarenkoMaxim/webapp-vanilla](https://github.com/ShinkarenkoMaxim/webapp-vanilla) - Telegram Web App starter bundle with vanilla setup.
```
Comment (roj1512) [inline README.md:46]:
```suggestion
- [ShinkarenkoMaxim/webapp-vanilla](https://github.com/ShinkarenkoMaxim/webapp-vanilla) - Telegram Web App starter template.
```

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

**Pull request merged**